client/components/wpadmin-auto-login/README.md

```diff
@@ -0,0 +1,13 @@
+WP-admin auto login
+===========
+
+This components opens a GET request to wp-admin of a connected site in order to log in to wp-admin.
+Because wp-login.php can automatically log in a user using SSO if `jetpack_sso_bypass_login_forward_wpcom` option is enabled, it redirects user automatically
+to a a transparent pixel.
+We use this component in Automated Transfer.
+
+## Requirements
+
+1. Site needs Jetpack enabled and working
```

Comment:
To be more specific, the Jetpack site needs to be connected to WordPress.com. Further, the site can not be in staging or development mode.